### PR TITLE
Convert app2 to use the users cookbook

### PR DESCRIPTION
**Pull request by TheREK3R**

### CODE REVIEW

File: recipes/app1.rb [hunk 1]
```diff
@@ -18,7 +18,17 @@
...
-node.default['users'] = %w(openid-staging openid-production)
+users = search('users', '*:*')
+
+users_manage 'app1' do
+  users users
+end
+
+users.each do |u|
+  directory "/home/#{u[:username] || u[:id]}" do
+    mode '02750'
```
Comment (ramereth) [inline recipes/app1.rb:29]: Is this what you were asking about the other day about what permission to set for this? Seems like this is something we should be able to configure in the data bag or have a better default in the upstream cookbook?

**Comment by TheREK3R** [comment]
oops need to revert this

**Review by ramereth — COMMENTED** [review]
> oops need to revert this

Re

**Comment by TheREK3R** [comment]
Shoot I think I reverted your jq push will redo now

**Review by ramereth — APPROVED** [review]

File: recipes/app3.rb [hunk 2]
```diff
@@ -90,14 +93,14 @@
...
-  end
-end
+# # Give nginx access to their homedirs
+# %w(production staging).each do |env|
+#   group "streamwebs-#{env}" do
+#     members ["streamwebs-#{env}", 'nginx']
+#     action :modify
+#     notifies :restart, 'service[nginx]'
+#   end
+# end
```
Comment (ramereth) [inline recipes/app3.rb:103]: What needs to happen with this commented code?

**Comment by ramereth** [comment]
!bump major ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a major-level version bump to v4.0.0.  Have a nice day!

**Pull request merged**